labs/b10.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Lab 10 - Version Control and Backups
+layout: lab
+---
+
+## Overview
+
+In this lab, we'll be talking about using version control, in particular Git, for keeping track of changes to your files and backing up your files to other machines. 
+We'll be setting up [Gitolite](http://gitolite.com/gitolite/index.html), an open-source Git server that allows you to host Git repositories on other machines,
+and set access to Git repositories hosted on the Gitolite server.
+
+## A Git Primer
+
+Version control tools are designed to keep track of changes to files, and allow you to rewind to some last saved state. 
+git, as one of the most popular version control tools in use today, supports these features, but also allows for more advanced workflows such as splitting a filebase into "branches" for experimentation.
+In order to complete this lab, you'll need to know a few basic git commands and some terminology.
+
+- A folder can be made into a git **repository** by running `git init`, telling git that files under the folder can start being tracked by git.
```

Comment:
Should probably explain that it does this by creating a `.git` directory in the root of the project.